ext/standard/basic_functions.h

```diff
@@ -196,6 +196,7 @@ typedef struct _php_basic_globals {
 	uint32_t *next;       /* next random value is computed from here */
 	int      left;        /* can *next++ this many times before reloading */
 
+    zend_bool mt_rand_seed_fixed; /* How mt_rand() has been seeded */
```

Comment:
This declaration should go below `mt_rand_is_seeded` to avoid ABI breakage (this will place it in padding bytes).